src/main/java/com/zufar/onlinestore/payment/service/PaymentService.java

```diff
@@ -0,0 +1,14 @@
+package com.zufar.onlinestore.payment.service;
+
+import com.stripe.exception.StripeException;
+import com.zufar.onlinestore.payment.dto.PaymentDetailsDto;
+import com.zufar.onlinestore.payment.dto.PaymentWithTokenDetailsDto;
+import java.math.BigDecimal;
+
+public interface PaymentService {
+
+    PaymentWithTokenDetailsDto createPayment(String paymentMethodId, BigDecimal totalPrice, String currency) throws StripeException;
+
+    PaymentDetailsDto getPayment(Long paymentId);
```

Comment:
add `throws PaymentNotFoundException` for better clarity please

Reply:
Done